src/main/java/de/zalando/aruha/nakadi/service/EventStreamConfig.java

```diff
@@ -7,6 +7,9 @@
 
 @Immutable
 public class EventStreamConfig {
+
+    private static int BATCH_FLUSH_TIMEOUT_DEFAULT = 30;
```

Comment:
Yes, I think it's a good idea to hide and resolve all default values inside builder. I'll try to do that.